samples/AuthRemoteFormsAuth/AuthRemoteFormsAuthCore/Program.cs

```diff
@@ -37,6 +39,14 @@
 app.MapControllerRoute(
     name: "default",
     pattern: "{controller=Home}/{action=Index}/{id?}");
-app.MapForwarder("/{**catch-all}", app.Configuration["ProxyTo"]!).WithOrder(int.MaxValue);
+
+// Configure the reverse proxy to forward all unhandled requests to the remote app
+app.MapForwarder("/{**catch-all}", app.Configuration["RemoteApp:Url"]!)
```

Comment:
NIT: If you want to also have HealthChecks enabled, you could also call here the `app.MapDefaultEndpoints()` from your ServiceDefaults project which should do that.